src/scaladoc/scala/tools/nsc/doc/html/page/JSON.scala

```diff
@@ -0,0 +1,71 @@
+package scala.tools.nsc.doc.html.page
+
+import JSONFormat.format
+
+object JSONFormat {
+
+  /**
+   * The default formatter used by the library. You can
+   * provide your own with the toString calls on
+   * JSONObject and JSONArray instances.
+   */
+  def format(x: Any): String = x match {
+    case s:  String     => s"""\"${quoteString(s)}\""""
+    case jo: JSONObject => jo.toString
+    case ja: JSONArray  => ja.toString
+    case other          => other.toString
```

Comment:
This is designed to dump all sorts of stuff unquoted into what is supposedly "JSON".  A strict warning is called for at the very least.